metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -366,6 +490,19 @@ def from_entry(cls, entry: AuditLogEntry) -> "QueryEvent":
 
         return queryEvent
 
+    @classmethod
+    def can_parse_exported_bigquery_audit_metadata(
+        cls, row: BigQueryAuditMetadata
+    ) -> bool:
+        try:
+            row["timestamp"]
+            row["protoPayload"]
+            metadata = json.loads(row["metadata"])
+            metadata["jobChange"]["job"]
+            return True
+        except (KeyError, TypeError):
+            return False
```

Comment:
As part of refactoring we have removed `can_parse_*` methods as they made debugging really hard of what is missing. Please see `get_missing_key_exported_bigquery_audit_metadata` which is just below this